src/test/java/staffconnect/storage/JsonAdaptedPersonTest.java

```diff
@@ -29,19 +30,31 @@ public class JsonAdaptedPersonTest {
     private static final String INVALID_TAG = "#friend";
     private static final String INVALID_AVAILABILITY = "m0n";
 
+    private static final String INVALID_MEETING_DESCRIPTION = "*!&@#&*@*&@*";
+
+    private static final String INVALID_MEETING_DATE = "12-04-2023 18:00";
+
     private static final String VALID_NAME = BENSON.getName().toString();
     private static final String VALID_PHONE = BENSON.getPhone().toString();
     private static final String VALID_EMAIL = BENSON.getEmail().toString();
     private static final String VALID_MODULE = BENSON.getModule().toString();
+    private static final String VALID_MEETING_DESCRIPTION = "Meet for midterms";
+
+    private static final String VALID_MEETING_DATE = "12-04-2023 18:00";
+
+    private static final JsonAdaptedMeeting VALID_MEETING_JSON =
+        new JsonAdaptedMeeting(VALID_MEETING_DESCRIPTION, VALID_MEETING_DATE);
+    private static final List<JsonAdaptedMeeting> VALID_MEETING =
+        Stream.of(VALID_MEETING_JSON).collect(Collectors.toList());
+
+    private static final List<JsonAdaptedTag> VALID_TAGS =
+        BENSON.getTags().stream().map(JsonAdaptedTag::new).collect(Collectors.toList());
```

Comment:
Any reason for the shift in code for `VALID_TAGS`?

Reply:
Shifted by merge